vector/src/main/java/im/vector/app/features/home/room/detail/composer/VoiceMessageRecorderView.kt

```diff
@@ -90,6 +96,9 @@ class VoiceMessageRecorderView @JvmOverloads constructor(
 
     override fun onVisibilityChanged(changedView: View, visibility: Int) {
         super.onVisibilityChanged(changedView, visibility)
+        // onVisibilityChanged is called by constructor on api 21 and 22.
+        if (!this::views.isInitialized) return
```

Comment:
@bmarty Were you able to reproduce the error on Android 5 and verify if this patch works?